apps/journeys-admin/src/components/DynamicPowerBiReport/Report/Report.tsx

```diff
@@ -19,9 +19,10 @@ export const GET_ADMIN_JOURNEYS_REPORT = gql`
 `
 export interface ReportProps {
   reportType: JourneysReportType
+  journeyId?: string // this is required for singleFull and singleSummary reports
 }
```

Comment:
Typescript can do this for us!

```suggestion
interface MultipleReportProps {
  reportType:
    | JourneysReportType.multipleFull
    | JourneysReportType.multipleSummary
  journeyId: undefined
}

interface SingleReportProps {
  reportType: JourneysReportType.singleFull | JourneysReportType.singleSummary
  journeyId: string
}

export type ReportProps = SingleReportProps | MultipleReportProps
```